### PR TITLE
In d insurance

### PR DESCRIPTION
**Pull request by gowthamd3**
---
When submitting a connector, please check the following conditions for your PR.

- [x] I attest that the connector works and I verfied by deploying and testing all the operations./
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

File: certified-connectors/IN-D Insurance/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,437 @@
+{
```
Comment (sriyen-msft) [inline certified-connectors/IN-D Insurance/apiDefinition.swagger.json:1]: @gowthamd3 This swagger is not up to date as per the artifacts you have provided in ISV Studio. Please remove the swagger and apiProperties.json files from the PR and add the connector artifacts you have provided in the ISV Studio.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/IN-D Insurance/apiDefinition.swagger.json (hunk 1 above)
Comment (gowthamd3) [reply certified-connectors/IN-D Insurance/apiDefinition.swagger.json:1]: Hi Srikanth, Updated the swagger file and apiProperties.json file.

**Review by sriyen-msft — DISMISSED** [review]

**Comment by sriyen-msft** [comment]
@gowthamd3 Please resolve the branch conflicts to get this PR merged

**Comment by gowthamd3** [comment]
> @gowthamd3 Please resolve the branch conflicts to get this PR merged

Hi Srikanth,

I have resolved the conflicts, this update is extension to previously merged #751 reference number PR, I think you are comparing with older version #560 reference no PR.

**Comment by sriyen-msft** [comment]
@gowthamd3 I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by sriyen-msft** [comment]
@gowthamd3 Since there are no updates for this PR, I am closing out and you can reopen it whenever you or your team is ready for pushing updates.

**Pull request closed without merging**